the-blue-alliance-ios/Core Data/CoreDataContextObserver.swift

```diff
@@ -77,7 +77,7 @@ public class CoreDataContextObserver<T: NSManagedObject> {
 
             for case let object as T in objectsSet {
                 guard let actionsForObject = actionsForManagedObjectID[object.objectID] else { continue }
-                actionsForObject.filter({ $0.state == state }).forEach({ $0.completionBlock(object, state) })
+                actionsForObject.filter({ $0.state == state || $0.state == .all }).forEach({ $0.completionBlock(object, state) })
```

Comment:
We could pull this out and move this in to a smaller PR.... I guess it doesn't matter much except for tracking